### ISSUE TITLE
String values get autocompleted with extra quotes

### ISSUE THREAD

**Issue by claviska** [break-stuff/cem-plugin-vs-code-custom-data-generator#10]
I'm reporting this issue to track it but, unfortunately, I don't have time today to jump in and fix it myself. Consider the following property defined in Lit:

```ts
/** The button's variant. */
@property({ reflect: true }) variant: 'default' | 'primary' | 'success' | 'neutral' | 'warning' | 'danger' | 'text' = 'default';
```

The resulting CEM data looks like this:

```json
"attributes": [
  {
    "name": "variant",
    "type": {
      "text": "'default' | 'primary' | 'success' | 'neutral' | 'warning' | 'danger' | 'text'"
    },
    "default": "'default'",
    "description": "The button's variant.",
    "fieldName": "variant"
  }
]
```

Note the extra single quotes shown in the `default` property. These end up in the autocomplete data:

https://user-images.githubusercontent.com/55639/203334782-6b51f67d-dc29-4c71-b610-5b96f4707f95.mp4

**Comment by break-stuff** [comment]
Good catch! I'll get a patch up shortly.

**Comment by break-stuff** [comment]
Resolved in release 1.2.1
